File: Python/Python Basics/Basic.py
# python - basic

# import external library
import keyword

# output operation
a = 10
print("hello",a)

# input operation
a = int(input("Enter a Number"))
# or 
b = input("Enter a Number")
print(a,"\t",b)

# multiple/split input
m,n = input("Enter 2 No").split()

# data type : Number, bool, string 

# type casting
integer = int('10')
floatp  = float(3.5)
strs    = str('string')

# operator
# unary ,binary ,ternary ,arithamatic ,logical ,relational 

# list of keywords in python
print(keyword.kwlist)

# delete data
del  a,b,m,n,integer,floatp,strs

File: Java/Java Basics/Basic.java

// java - Basic
import java.util.Scanner;

class JavaBasic {
    public static void main(String[] args) {
        // printing
        // println() for new line otherwise print()
        System.out.print("HELLO");

        int number; // int variable as sample
        Scanner sc = new Scanner(System.in); // initialise scanner

        System.out.println("ENTER A NUMBER"); // ask for integer input

        // store input in variable
        number = sc.nextInt();

        System.out.println(number);

        // data types :
        // int ,char ,float ,double ,long ,boolean
        // string ,array

        int i = 10;
        long l = 2;
        float f = 3.14F;
        double d = 100.25D;

        System.out.println(i + l + f + d);

        String name = "";
        int[] array = { 1, 2, 3 };

        System.out.println(name + array[0]);

        // operator :
        // unary ,binary ,trinary
        // arithamatic ,logical ,assignment ,bitwise ,relational
        // operator works same in all language

        // type casting
        // double to int
        double num = 10;
        int num2 = (int) num;
        System.out.println("double => int " + num2);
        sc.close();
    }
}